docs/code_review.md

```diff
@@ -54,3 +54,6 @@
   rules noted in [Tracking of workarounds](./workarounds.md) page are
   followed. The person merging a workaround is responsible for checking that all
   information related to the workaround is correct.
+* MCG test cases should work on any cloud platform (AWS, Azure, GCP, ...).
+  Validation runs of MCG pull request should include runs of at least two such
```

Comment:
even though we have seen in the past tests that passed over AWS but not over Azure (namespace bucket), I would say that we should verify on 1 cloud based and 1 on prem based

Reply:
Hmm, right. I understand that running validation job on multiple platforms is an overhead.

Reply:
I just dropped this line.